edalize/rivierapro.py

```diff
@@ -24,6 +24,9 @@ def get_doc(cls, api_ver):
                         {'name' : 'vsim_options',
                          'type' : 'String',
                          'desc' : 'Additional run options for vsim'},
+                        {'name' : 'compilation_mode',
+                         'type' : 'String',
+                         'desc' : 'Common or separate compilation, sep - for separate compilation, common - for common compilation'},
                         ]}
```

Comment:
This parameter should not be in the `lists` section, but in a section called `members`. See e.g. [here](https://github.com/olofk/edalize/blob/master/edalize/verilator.py#L43) for a backend that uses both lists and members.

Also, when this change is made, the check for them will be `if (self.tool_options.get('compilation_mode'))=='sep'` instead of `if (self.tool_options.get('compilation_mode'))==['sep']`